src/status_im/ui/screens/profile/user/styles.cljs

```diff
@@ -1,6 +1,80 @@
-(ns status-im.ui.screens.profile.user.styles)
+(ns status-im.ui.screens.profile.user.styles
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.safe-area :as safe-area]))
 
 (def share-link-button
   {:margin-top        12
    :margin-horizontal 16
    :margin-bottom     16})
+
+(def radius 16)
+
+(def top-background-view
+  {:background-color colors/magenta-opa-40
+   :position         :absolute
+   :top              0
+   :left             0
+   :height           400
+   :width            400
+   :z-index          -1})
+
+(def toolbar
+  {:padding-bottom  16
+   :padding-top     (safe-area/get-top)
```

Comment:
@smohamedjavid this is alright to do, or we normally get this value at the rendering of the component view in a let var? 🤔

Reply:
You are right. We usually get the safe area value from the component's let var. 👍 

```clojure

;; view.cljs
(let [top (safe-area/get-top)]
  [rn/view {:style (style/toolbar top)}])

;; style.cljs
(defn toolbar
  [top]
  {:padding-bottom  16
   :padding-top     top
   :flex-direction  :row
   :align-items     :center
   :justify-content :space-between})
```